v-next/hardhat/src/internal/cli/main.ts

```diff
@@ -378,7 +378,7 @@ function parseDoubleDashArgs(
   cliArguments: string[],
   usedCliArguments: boolean[],
   optionsMap: Map<string, TaskOption | GlobalOption>,
-  argumentsMap: TaskArguments,
+  argumentsMap: TaskArguments | Partial<GlobalOptions>,
```

Comment:
I've been out the last few days, so I might have missed some context. However, the naming here seems a bit odd now with the rename to "options".

Reply:
the rename is still incomplete, I added a few missing things in #5421